packages/wicked/wicked.spec

```diff
@@ -26,6 +26,13 @@ Source12: common.xml
 Source13: nanny.xml
 Source14: server.xml
 
+# Override service units to make it easier to pass new options.
+Source20: wicked.service
+Source21: wickedd.service
+Source22: wickedd-dhcp4.service
+Source23: wickedd-dhcp6.service
+Source24: wickedd-nanny.service
+
 %if %{without bootstrap}
 Source99: constants.xml
 %endif
```

Comment:
If we don't use auto4, can we also remove `autoip4` in `0002-exclude-unused-components.patch`?